src/rgw/rgw_sal_motr.cc

```diff
@@ -2711,6 +2724,7 @@ int MotrMultipartUpload::init(const DoutPrefixProvider *dpp, optional_yield y,
 
     motr_multipart_upload_info upload_info;
     upload_info.dest_placement = dest_placement;
+    upload_info.upload_id = upload_id;
```

Comment:
Also do we need to add update upload_id in MotrMultipartUpload::get_info() function as well?

Reply:
get_info only needs placement and etag information and doesnt expect upload_id. Only selective information is fetched and provided in this function.